engine-tests/src/test/java/org/terasology/math/ChunkMathTest.java

```diff
@@ -93,4 +93,14 @@ public void testCalcChunkPos() {
         assertTrue(ChunkMath.calcChunkPos(10, 10, 10, temp).equals(0, 0, 0));
     }
 
+
+    @Test
+    public void testFloatingPointCalcChunkPos() {
+        org.joml.Vector3i temp = new org.joml.Vector3i();
+        assertTrue(ChunkMath.calcChunkPos(31.9f, 64.1f, 32.5f, temp).equals(0, 1, 1), temp.toString());
+        assertTrue(ChunkMath.calcChunkPos(32.9f, 63.9f, 32.9f, temp).equals(1, 0, 1), temp.toString());
```

Comment:
wait what ... ohhh, chunks aren't cubical, they're taller than they are wide? okay. then my previous comment about where the x-boundary for chunk -1 is was wrong.